src/main/java/seedu/address/storage/XmlAddressBookStorage.java

```diff
@@ -76,7 +76,7 @@ public void saveAddressBook(ReadOnlyAddressBook addressBook, String filePath) th
     }
 
     public void backupAddressBook(ReadOnlyAddressBook addressBook) throws IOException {
-        saveAddressBook(addressBook, filePath.concat("backup.fxml"));
+        saveAddressBook(addressBook, "backup.fxml");
```

Comment:
Why not include "filePath"?

Reply:
As mentioned in the comments by tutor, we can have a constant filepath instead of a dynamic one.

Reply:
Ok, I understand.